src/main/java/org/launchcode/liftoffproject/models/Chore.java

```diff
@@ -15,6 +19,9 @@ public class Chore extends AbstractEntity{
     @ManyToOne
     private Child childAssigned;
 
+    @NotBlank
```

Comment:
```suggestion
```
Think we should remove the `@NotBlank` annotation and not require the detailed description. That way if the chore is something simple where the detailed description is semi-pointless we don't make people have to include it.